File: app/src/main/java/com/allenliu/btdemo/WelcomeActivity.java
package com.allenliu.btdemo;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class WelcomeActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_welcome);
        new Handler().postDelayed(()->{
            startActivities(new Intent[]{new Intent().setClass(this, MainActivity.class)});
        },3500);
    }
}

File: app/src/main/java/com/allenliu/btdemo/WelcomeActivity.kt
package com.allenliu.btdemo

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity

class WelcomeActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_welcome)

        Handler().postDelayed({
            startActivity(Intent().setClass(this,MainActivity().javaClass))
        }, 3500)
    }
}
